src/lib/stitching/gravity/stitching.ts

```diff
@@ -197,6 +202,23 @@ export const gravityStitchingEnvironment = (
             return formatMarkdownValue(description, desiredFormat)
           },
         },
+        artworksCountMessage: {
+          fragment: gql`
+            ... on ArtistSeries {
+              forSaleArtworksCount
+              artworksCount
+            }
+          `,
+          resolve: async ({ forSaleArtworksCount, artworksCount }) => {
+            const artworksCountMessage = forSaleArtworksCount
+              ? `${forSaleArtworksCount} available`
```

Comment:
**suggestion (non-blocking):** Reformat as explicit `if/else` statements since the nested ternaries make it difficult to quickly grasp what's going on. Non-blocking mostly because it's still clear, just requires a little more effort.

Reply:
Good call

Reply:
Fixed in https://github.com/artsy/metaphysics/commit/668c3733457c6075c765deeed9c6ff22754418ca#diff-573ae00d2f637334b78f6df4edf48571R215-R221